sys/compat/linux/linux_ioctl.c

```diff
@@ -2219,7 +2220,7 @@ linux_ifconf(struct thread *td, struct ifconf *uifc)
 				valid_len = sbuf_len(sb);
 		}
 		if (addrs == 0) {
-			bzero((caddr_t)&ifr.ifr_addr, sizeof(ifr.ifr_addr));
+			bzero(&ifr.ifr_ifru.ifru_addr, sizeof(ifr.ifr_ifru.ifru_addr));
```

Comment:
Should this not be `ifr_addr_get_sa` too to get the right compat one?

Reply:
This one isn't actually a `struct ifreq`.  It's a `struct l_ifreq` and the code abuses the macros to access members of the union that we don't enable in kernel in order to force use of the accessors.  Hmm, this code could probably go away since it's 256-bit related (you need to compensate for the 16-bytes of padding)...